scripts/home.js

```diff
@@ -126,25 +126,38 @@ const exchangeMoney = async e => {
 	}
 };
 
-const showTransactionDetails = () => {
+const showTransactionDetails = async () => {
 	const arsAmount = document.getElementById("exchangeAmount").value;
 	const exchangeRate = document.getElementById("exchangeRate");
 	const usdAmount = document.getElementById("usdAmount");
 	const exchangeTax = document.getElementById("exchangeTax");
 	const totalAmount = document.getElementById("exchangeTotal");
 
-	getExangeRate().then(response => (exchangeRate.textContent = response.US$));
-	exchangeTax.textContent = +arsAmount * 0.3;
+	await getExangeRate().then(response => (exchangeRate.textContent = response.US$));
+	if (exchangeRate.textContent === "N/A") {
+		exchangeTax.textContent = "N/A";
+		usdAmount.textContent = "N/A";
+		totalAmount.textContent = "N/A";
+	} else {
+		exchangeTax.textContent = (+arsAmount * 0.3).toFixed(2);
```

Comment:
this is weird jajaja
más allá del await y el .then en la misma línea, que onda la verificación al string N/A?? por ahora es funcional y va, pero a chequear